gallifrey/backend/src/main/java/gallifrey/backend/VectorClockBackend.java

```diff
@@ -185,6 +182,37 @@ public Object rmiOperation(GenericKey key, GenericFunction func) throws RemoteEx
         return temp_crdt_object.invoke(func);
     }
 
+    @Override
+    public Object rmiOperation(GenericKey key, GenericFunction func) throws RemoteException, BackendRequiresFlushException {
+        VectorClock currentDownstreamTime = LastDownstreamTime;
+
+        if (LastUpdateTime.lessthan(currentDownstreamTime)) {
```

Comment:
sorry I just realized this isn't quite right, we want LastUpdateTime to be after currentDownstreamTime.  So you were looking for: 

`!currentDownstreamTime.lessThan(LastUpdateTime)`